sdk/storage/src/blobs/block_blob_client.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "blobs/block_blob_client.hpp"
+#include "common/storage_common.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  BlockBlobClient::BlockBlobClient(const std::string& blobUri, BlockBlobClientOptions options)
+      : BlobClient(blobUri, options)
+  {
+    unused(options);
+  }
+
+  BlockBlobClient BlockBlobClient::WithSnapshot(const std::string& snapshot)
+  {
+    BlockBlobClient new_client(*this);
+    new_client.m_blobUrl += "?snapshot=" + snapshot;
+    return new_client;
+  }
+
+  BlobContentInfo BlockBlobClient::Upload(
+        // TODO: We don't have BodyStream for now.
+        std::vector<uint8_t> content,
+        const UploadBlobOptions& options)
+  {
+    BlobRestClient::BlockBlob::UploadOptions protocol_layer_options;
```

Comment:
variables should use modified 'pascalCase'

Reply:
fixed